app/templates/app/search.html

```diff
@@ -6,81 +6,161 @@
+                                <li>
+                                    <span class="mr-5"><b>{% trans "Mime Type:" %}</b></span>
+                                    <span>{{ document.mime_type }}</span>
+                                </li>
+                                <li class="mt-2">
+                                    <span class="mr-5"><b>{% trans "Rank:" %}</b></span>
+                                    <span><i>{{ document.rank }}</i></span>
+                                </li>
+                            </ul>
+                        </div>
+                    {% endfor %}
+                </div>
+            </div>
+            <div class="container">
+                <hr>
+                <div class="pagination ">
+                    {% if documents.has_previous %}
+                        <a href="?search={{ request.GET.search|default_if_none:'' }}
+                    &document_type={{ request.GET.document_type|default_if_none:'' }}
+                    &institution={{ request.GET.institution|default_if_none:'' }}
+                    &subjects={{ request.GET.subjects|default_if_none:'' }}
+                    &languages={{ request.GET.languages|default_if_none:'' }}&page=1">&laquo; first</a>
+                        <a href="?page={{ documents.previous_page_number }}">&lt; previous</a>
+                    {% endif %}
+                    <span class="current">
+                          Page {{ documents.number }} of {{ documents.paginator.num_pages }}.
+                    </span>
+                    {% if documents.has_next %}
+                        <a href="?search={{ request.GET.search|default_if_none:'' }}
+                    &document_type={{ request.GET.document_type|default_if_none:'' }}
+                    &institution={{ request.GET.institution|default_if_none:'' }}
+                    &subjects={{ request.GET.subjects|default_if_none:'' }}
+                    &languages={{ request.GET.languages|default_if_none:'' }}
+                    &page={{ documents.next_page_number }}">next &gt;</a>
+                        <a href="?search=
+                            {{ request.GET.search|default_if_none:'' }}&page={{ documents.paginator.num_pages }}">last
+                            &raquo;</a>
+                    {% endif %}
+                </div>
+                <hr>
+            </div>
+            <form method="get" class="container">
+                <input type="search" name="{{ filter.form.search.name }}" placeholder="{% trans 'Search...' %}"
+                       class="form-control search-input"
+                       value="{{ request.GET.search|default_if_none:'' }}">
+                <br>
+                <div class="row">
+                    <div class="col-md-3">
+                        <div class="form-group">
+                            <label for="id_subjects">Institution</label>
+                            <div class="subjects-checkbox-container">
```

Comment:
Please consider a more general name for the container, as this is not about subjects.